tests/TestManager.cfc

```diff
@@ -0,0 +1,116 @@
+component extends = "mxunit.framework.TestCase" {
+
+	function beforeTests() {
+		try {
+			variables.cacheManager = new lib.ehcache.Manager();
+		} catch(Any e) {
+			variables.exception = e;
+		}
+	}
+
+	function afterTests() {
+		variables.cacheManager.getCache("mxunitCache").flush();
+//		variables.cacheManager.removeCache("mxunitCache");
```

Comment:
@timmixell do we need these commented lines anymore? Or keep them for later use?